docker/scripts/reloader.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Can't we have a default config according to our needs instead of having a config file?

Reply:
I agree path may differ among projects

Reply:
The logic behind is two-fold:
* I didn't want to parse (too much) the command-line, I just a quick&forget feature to get rid of that infuriating pile of garbage that is nodemon
* this makes more apparent that a configuration exists and that this tool can be configured, think of this as a documentation of a sort

But if you think that it's far better to remove the configuration file, I can of course do it, I guess.

Reply:
I removed the configuration file. If we need this tool to be configurable, we can work on it later.

While I was at it, I added a watcher on typescript files in `docker/scripts` so changes on the functional-test script or on the starter one now trigger a restart.